nixpkgs/nixpkgs.bzl

```diff
@@ -94,7 +104,7 @@ def _nixpkgs_package_impl(ctx):
   timeout = 1073741824
 
   res = ctx.execute(nix_build, quiet = False, timeout = timeout,
-                    environment=dict(NIX_PATH="nixpkgs=" + nix_path))
+                    environment=dict(NIX_PATH=nix_path))
```

Comment:
This breaks the `path` argument, since it’s not of the form `"nixpkgs=path"` anymore.